powerquery-docs/Connectors/SoftoneBI.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Softone BI connector
```

Comment:
Should this be "SoftOne BI" to be consistent with your connector Publish

Reply:
Yes, that is correct.